app/docker/components/datatables/containers-datatable/containersDatatable.html

```diff
@@ -161,29 +215,32 @@
                 </span>
                 <a ui-sref="docker.containers.container({ id: item.Id, nodeName: item.NodeName })" title="{{ item | containername }}">{{ item | containername | truncate: $ctrl.settings.containerNameTruncateSize }}</a>
               </td>
-              <td>
+              <td ng-show="$ctrl.columnVisibility.columns.state.display">
                 <span ng-if="['starting','healthy','unhealthy'].indexOf(item.Status) !== -1" class="label label-{{ item.Status|containerstatusbadge }} interactive" uib-tooltip="This container has a health check">{{ item.Status }}</span>
                 <span ng-if="['starting','healthy','unhealthy'].indexOf(item.Status) === -1" class="label label-{{ item.Status|containerstatusbadge }}">{{ item.Status }}</span>
               </td>
-              <td ng-if="$ctrl.settings.showQuickActionStats || $ctrl.settings.showQuickActionLogs || $ctrl.settings.showQuickActionConsole || $ctrl.settings.showQuickActionInspect">
+              <th ng-show="$ctrl.columnVisibility.columns.created.display">
```

Comment:
You'll want to use `td` instead of `th` tags here.

Also, please use the `getisodatefromtimestamp` filter instead of `getisodatefromtimestamp`.